app/forms/life_situations_form.rb

```diff
@@ -1,7 +1,20 @@
 class LifeSituationsForm < QuestionsForm
   set_attributes_for :intake, :was_full_time_student, :was_on_visa, :had_disability, :was_blind
+  set_attributes_for :misc, :no_life_situations_apply
+  validate :at_least_one_option_selected
 
   def save
     @intake.update(attributes_for(:intake))
   end
+
+  private
+
+  def at_least_one_option_selected
+    chose_one = was_full_time_student == "yes" ||
+      was_on_visa == "yes" ||
+      had_disability == "yes" ||
+      was_blind == "yes" ||
+      no_life_situations_apply == "yes"
+    errors.add(:no_life_situations_apply, I18n.t("forms.errors.at_least_one_option")) unless chose_one
```

Comment:
we've tried to standardize on a patern for this where you implement `at_least_one_selected` to be true/false and the `errors.add` is done by a `at_least_one_or_none_of_the_above_selected` validator

see https://github.com/codeforamerica/vita-min/blob/main/app/forms/ctc/dependents/relative_qualifiers_form.rb#L8 https://github.com/codeforamerica/vita-min/blob/main/app/validators/at_least_one_or_none_of_the_above_selected_validator.rb